doc/building.md

```diff
@@ -115,8 +125,10 @@ You can install both compilers and select the one to be used by `./configure`.
 
  - (obviously) clone this repo or acquire the sources some other way
  - make sure you have everything from the [dependency list](#dependencies)
- - `./configure --compiler=clang` will prepare building
+ - For *nix: `./configure --compiler=clang` will prepare building
   - You could also use `./configure --mode=release --compiler=gcc` here
+ - For Mac OS X: use instead:  `./configure --cpp-compiler=clang++-3.5`
+  -  Or: `./configure --mode=release --compiler=g++-4.9`
```

Comment:
Brew  installs these compilers with the version number as a part of the
executable name, so that they do not not interfere with already-installed
Apple tools. Brew doesn't create symlinks without the version number
(unlike Linux).

If someone uses `--compiler=clang` then it would run the Apple-installed
clang and fail. The same would happen with gcc.

--

Karlsruhe Institute of Technology (KIT)
Institute of Theoretical Informatics

Dr. Darren Strash
Postdoctoral Researcher

Am Fasanengarten 5
Building 50.34, Room 206
76131 Karlsruhe

Telephone: +49 721 608-4-6602
E-Mail: strash@kit.edu

KIT - Universität des Landes Baden-Württemberg und nationales
Großforschungszentrum in der Helmholtz-Gemeinschaft
On Sep 18, 2015 11:47 PM, "Jonas Jelten" <notifications@github.com> wrote:

> In doc/building.md
> <https://github.com/SFTtech/openage/pull/388#discussion_r39904507>:
>
> >    - You could also use `./configure --mode=release --compiler=gcc` here
> > + - For Mac OS X: use instead:  `./configure --cpp-compiler=clang++-3.5`
> > +  -  Or: `./configure --mode=release --compiler=g++-4.9`
>
> What happens for mac users if you chose cpp-compiler=clang or clang++?
> Why is the explicit version required?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SFTtech/openage/pull/388/files#r39904507>.
>